src/interactivity/Interactivity.js

```diff
@@ -167,13 +169,32 @@ export default class Interactivity {
         });
     }
 
+    _subscribeToLayerEvents (layers) {
+        layers.forEach(layer => {
+            // Debounce `updated` event calls
+            // - Delay: 60 ms
+            // - Maximum delay: 100 ms
+            layer.on('updated', _.debounce(this._onLayerUpdated.bind(this), 60, { maxWait: 100 }));
```

Comment:
As I wrote in the outdated comment a debounce is not necessary, in fact, I think we shouldn't use one. I proposed to throw an event at a maximum rate of one per frame, (coalescing all the `updated` events of the same frame).

I don't think we should use a debounce because it's the application code the one who should be responsible for throttling to avoid performance issues related to intensive processing in response to the event. Some applications will need major throttling but others will need less or none at all. We are also having this behavior on the `updated` event which is part of the public API and is not throttled.

Reply:
OK, you've convinced me. Let's trigger at the maximum rate and let the users optimize the performance if required.

Reply:
Done